### DISCUSSION TITLE
New user disconnects out admin

### DISCUSSION THREAD

**Discussion by alwerr** [pocketbase/pocketbase#5534] [Q&A]
So, in my flutter app i put this in main.dart:
`void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await Storage().init();
  pb = PocketBase('http://url/',authStore:  AsyncAuthStore(
    save:    (String data) async => Storage.set('pb_auth', data),
    initial: Storage.get('pb_auth'),
  ));
  try {
    var admin = await pb.admins.authWithPassword('admin@admin.com', 'password');
    runApp(const MyApp());
  } catch (e){
    print(e);
  }
}`
And everything works. When signing in new user :
`      pb.collection('users').authWithPassword()`
The new user created but admin discoonect, so when i navigate the app im getting the error:
Only admins can perform this action.
Is tthere a way to fix it?
Thanks, Pocketbase is great!

**Comment by ganigeorgiev** [comment]
The "disconnect" happens because `pb.authStore` is shared no matter whether it is used with regular users or admins (*with the next release admins are also converted to special auth collection named "_superusers"*).

It is not clear what actually you are trying to do but **usually you shouldn't hardcode the admin credentials as part of a client-side app as it will be insecure and everyone will be able to read them**. Keep in mind that admins can access and modify everything, including deleting other users or even all collections.

If for some reason you really want to have both admin and regular user auth as part of your app then you can create a new separate Dart SDK instance for your regular users (make sure that they use different storage key).

If the above doesn't help, please elaborate a little more on your use case to understand the bigger picture and what actually you are trying to do.
